index.html

```diff
@@ -1,200 +1,210 @@
 <html lang="en">
+
 <head>
-	<title>HealthMove</title>
-	<link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">
-	<link rel="stylesheet" href="theme/fonts/Quicksand">
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-	<link rel="stylesheet" href="theme/build/styles.min.css">
-	<script src="theme/node_modules/swiper/dist/js/swiper.min.js"></script>
-	<link rel="stylesheet" href="theme/css/swiper.css">
-	<link rel="stylesheet" href="theme/node_modules/swiper/dist/css/swiper.min.css">
-	<script src="theme/build/script.min.js"></script>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HealthMove</title>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="theme/fonts/Quicksand">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="theme/build/styles.min.css">
+    <script src="theme/node_modules/swiper/dist/js/swiper.min.js"></script>
+    <link rel="stylesheet" href="theme/node_modules/swiper/dist/css/swiper.min.css">
+    <script src="theme/build/scripts.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="dist/hamburgers.css" rel="stylesheet">
```

Comment:
I can't see a folder for this hamburgers stylesheet... should this line still be here?

Reply:
yes it should. The folder is in node_modules

Reply:
I can't see the dist folder but i see that the library is being used in your header, it doesn't seem like you need to include this line here if you're including the sass file from the node modules in the app.scss.

Reply:
I've looked at the library you're using, this line definitely does not need to be here since you install the library with npm and NOT manually which is when this line would be suitable.

Reply:
my mistake. Changing now. Thank you.

Reply:
do you remember what the line above is from?? I don't think I need that either.